cirq-google/cirq_google/transformers/__init__.py

```diff
@@ -0,0 +1,18 @@
+# Copyright 2022 The Cirq Developers
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+from cirq_google.transformers.analytical_decompositions import (
+    known_2q_op_to_sycamore_operations,
```

Comment:
Is this the same situation we had w/ iswap where we had one method that would work for parameterized case and one that wouldn't ?

Reply:
`known_2q_op_to_sycamore_operations` still doesn't work for parameterized cases but I believe we can make it work. The main difference here is that since we don't have a direct analytical decomposer using KAK to SYC gates, we have some pre-computed optimal decompositions as part of `known_2q_op_to_sycamore_operations` and if they are not enough, we fall back to first converting arbitrary unitary to CZs and then further decomposing each CZ into SYCs, which happens in `two_qubit_matrix_to_sycamore_operations`.